build/build/Cargo.toml

```diff
@@ -19,7 +19,7 @@ cfg-if = "1.0.0"
 chrono = { version = "0.4.19", features = ["serde"] }
 clap = { version = "3.1.5", features = ["derive", "env", "wrap_help"] }
 derivative = "2.2.0"
-derive_more = "0.99.17"
+derive_more = "0.99"
```

Comment:
Why this change? Is 0.99.17 known to break something? Then maybe we should add more strict constraints?

Reply:
We depend on `derive_more` in multiple crates, in some we referred to it with strict version, in others it was just `0.99`. This change ensures that the same version will be used across the board. In this case it isn't that important, but with other crates (like `syn` or `quote`) having the versions specified differently (even if the same is used in the end) can lead to trait resolution errors.